sql/core/src/main/scala/org/apache/spark/sql/SQLConf.scala

```diff
@@ -138,7 +139,13 @@ private[sql] trait SQLConf {
    * When set to true, we always treat byte arrays in Parquet files as strings.
    */
   private[spark] def isParquetBinaryAsString: Boolean =
-    getConf(PARQUET_BINARY_AS_STRING, "false").toBoolean
+    getConf(PARQUET_BINARY_AS_STRING, "true").toBoolean
```

Comment:
Oh, this is a mistake... I'll modify it in next commit. Actually what I meant to change is the following value.
Thanks!